ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -473,6 +484,16 @@ export const WorkspaceAbout = fp.flow(
                 </StyledExternalLink>
               </div>
             </TooltipTrigger>
+            <div>
+              <h3 style={{ marginBottom: '0.5rem' }}>File Management</h3>
```

Comment:
If the workspace doesn't exist (as indicated by the `this.state.workspace ?` check above, should we still show this section? I may be misunderstanding the behavior of `this.state.workspace`.

Reply:
That is added in workspace about page. That workspace has to exist to enter this page

Reply:
Ok, maybe there's a way to avoid checking for `state` existence on this page? I think that may be out of the scope for this PR though.